apps/web/src/pages/studies/[studyId]/index.tsx

```diff
@@ -172,11 +192,16 @@ export const getServerSideProps = withServerSideProps(Roles.SponsorContact, asyn
     }
   }
 
+  const userEmail = session.user?.email || ''
+
+  const editStudyAllowList = process.env.EDIT_STUDY_ALLOW_LIST?.split(',') || []
```

Comment:
As above, should this be DB state?
e.g. driven from a user role?

Reply:
Addressed above 👍